src/snowflake/cli/api/project/schemas/project_definition.py

```diff
@@ -200,6 +220,18 @@ def apply_mixins(cls, data: Dict, info: ValidationInfo) -> Dict:
                     mixin_defs=data["mixins"],
                 )
                 entities[entity_name] = merged_values
+                if entity["type"] == "application package" and "children" in entity:
+                    for i, child in enumerate(entity["children"]):
+                        child_mixins = entity_mixins_to_list(child.get("use_mixins"))
+                        merged_values = cls._merge_mixins_with_entity(
+                            entity_id=entity_name,
+                            entity=child,
+                            entity_mixins_names=child_mixins,
+                            mixin_defs=data["mixins"],
+                            allow_any=True,
```

Comment:
this feels a bit off maybe? It sounds like in the context of children we're just completely disabling any kind of field checks. I get why you'd do it for a POC, but we should probably be careful about allowing that.

Reply:
Removed mixins support for this PR